helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -42,6 +50,26 @@ receivers:
     - reason: FailedCreate
       involvedObjectKind: Job
   {{- end }}
+  {{- if eq (include "splunk-otel-collector.distribution" .) "eks/fargate" }}
+  # dynamically created kubeletstats receiver to report all Fargate "node" kubelet stats
+  # with exception of collector "node's" own since Fargate forbids connection.
```

Comment:
Does signalfx-agent has the same issue? I see it has only one deployment.

Reply:
Yes, this is a general fargate and eks restriction (https://github.com/signalfx/signalfx-agent/tree/main/deployments/k8s#aws-eksfargate) and the eks fargate deployment is limited: no gateway/forwarder support and no self-pod/node metrics.